scripts/2022_06_14_19m_sdl_to_jump.py

```diff
@@ -0,0 +1,37 @@
+from helpers import CHAIN_IDS, MULTISIG_ADDRESSES, SDL_ADDRESSES, JUMP_RECEIVER_ADDRESS
+from ape_safe import ApeSafe
+from brownie import accounts, network
+
+from scripts.utils import confirm_posting_transaction
+
+
+def main():
+    """Send 19,999,999 $SDL to Jump"""
+
+    print(f"You are using the '{network.show_active()}' network")
+    deployer = accounts.load("deployer")  # prompts for password
+    multisig = ApeSafe(MULTISIG_ADDRESSES[CHAIN_IDS["MAINNET"]])
+
+    jump_receiver = JUMP_RECEIVER_ADDRESS[CHAIN_IDS["MAINNET"]]
+    sdl_contract = multisig.contract(SDL_ADDRESSES[CHAIN_IDS["MAINNET"]])
+    sdl_vesting_contract = multisig.contract(
+        "0xf8504e92428d65E56e495684A38f679C1B1DC30b"
```

Comment:
Could add this to the consts file since we'll probably be claiming from this again in the future